Extend FAQ (#65)
saucelabs/testrunner-toolkit: docs/FAQS.md

Below are list of frequently asked questions regarding some of the business and technical use cases surrounding the Testrunner Toolkit.

<!--__Table of Contents__
* [Why Do I See This Error?](#why-do-i-see-this-error?)
* [Do My Tests Run In The Sauce Labs Cloud?](#do-my-tests-run-in-the-sauce-labs-cloud?)
* [Why Testrunner Toolkit?](#why-testrunner-toolkit?)
* [Where are the Docker Images and Can I Inspect Them?](#where-are-the-docker-images-and-can-i-inspect-them?)
* [Are There any Framework Test Code Samples?](#are-there-any-framework-test-code-samples?)
* [How Do I Use This in my Pipeline?](#how-do-i-use-this-in-my-pipeline?)
-->

## Why Do I See This Error?

> If you see a message that looks like `Misconfigured -- Unsupported OS/browser/version/device combo: OS: '*', Browser: 'googlechrome', Version: '*.', Device: 'unspecified'` in your Job's metadata, rest assured that your Job has appropriately succeeded or failed, depending on the test results themselves and independent of this error message. It is an artifact of our early beta and will be addressed soon.

Reviewer: ```suggestion
If you see the following message in your Job's metadata: `Misconfigured -- Unsupported OS/browser/version/device combo: OS: '*', Browser: 'googlechrome', Version: '*.', Device: 'unspecified'`, it is an artifact of our early beta and will be addressed soon. Rest assured that your Job should appropriately succeed or fail depending on the test results themselves and in spite of this error message.
```
